tests/helpers/runif.py

```diff
@@ -195,6 +197,13 @@ def __new__(
             conditions.append(not _OMEGACONF_AVAILABLE)
             reasons.append("omegaconf")
 
+        if slow:
```

Comment:
a job cannot be marked as slow and require gpu. should we handle this case? There may be other collisions which might be annoying.

Reply:
We can do a simple `if` check for it here if both are passed together, but it won't work with:

```python
@RunIf(min_gpus=1)
@RunIf(slow=True)
def test_something():
    ...

# or, more common pattern

@RunIf(slow=True)
@pytest.mark.parametrize("an_arg", [False, pytest.param(True, marks=RunIf(min_gpus=1))])
def test_something(an_arg):
    ...
```